test/sample/test_sample.py

```diff
@@ -836,3 +856,90 @@ def test_thermodynamic_integration():
     assert np.isclose(log_evidence, np.log(evidence[0]), atol=tol)
     assert np.isclose(log_evidence_not_all, np.log(evidence[0]), atol=tol)
     assert np.isclose(log_evidence_simps, np.log(evidence[0]), atol=tol)
+    assert np.isclose(
+        log_evidence_steppingstone, np.log(evidence[0]), atol=tol
+    )
+
+
+@pytest.mark.flaky(reruns=2)
+def test_harmonic_mean_log_evidence():
+    tol = 1
+    # define problem
+    problem = gaussian_problem()
+
+    # run optimization and MCMC
+    result = optimize.minimize(
+        problem,
+        progress_bar=False,
```

Comment:
```suggestion
        progress_bar=False,
        n_starts=10,
```
See comment above or below